compose.yaml

```diff
@@ -5,15 +5,20 @@ services:
     init: true
     build:
       context: .
+      target: development
```

Comment:
:thought_balloon: The purpose of this configuration is to build for production, not development, as the default. There's another problem in that all of the `bin/docker` scripts are meant for development while this configuration is for production. That's the split.

Reply:
Addressing in response to your larger PR comment